### DISCUSSION TITLE
Recreate codespace from prebuilt image.

### DISCUSSION THREAD

**Discussion by Aubron** [community/community#16865] [Codespaces]
As far as I can tell, you cannot combine prebuilt images and persistent codespaces in a way that I would like to.

If I have a prebuilt image from the commit, I would like the capability to recreate the container from that prebuilt image.

Today I can only "Rebuild" the image, a useful feature if I am working on the container configuration, but not if I am just wanting to solve a workspace quirk or configuration drift.

**Comment by Aubron** [comment]
The workaround is to create a new codespace, but I have frustratingly lost workspace changes as a result of this, not thinking about the fact that a new codespace will have a new working copy.

**Comment by dmilburn** [comment]
Hi from the Codespaces team! Thanks for sharing this feedback and I'm sorry for your friction. I've shared it with the team as a feature request
